lib/SmartyPlugins/block.box_header.php

```diff
@@ -41,8 +41,8 @@ function smarty_block_box_header($params, $content, $template, $repeat)
             return '
 		<div' . ($id ? ' id="' . $id . '"' : '') . ' class="lms-ui-box-header">
 		' . (strpos($icon, '/') !== false ? '<IMG src="' . $icon . '" alt="">'
-                    : (strpos($icon, 'lms-ui-icon-') === 0 ? '<i class="' . $icon : '<i class="lms-ui-icon-' . $icon))
-                    . (!empty($icon_class) ? ' ' . $icon_class : '') . '"></i>'
+                    : (strpos($icon, 'lms-ui-icon-') === 0 ? '<i class="' . $icon : '<i class="lms-ui-icon-' . $icon . '">'))
+                    . (!empty($icon_class) ? ' ' . $icon_class : '') . '</i>'
                   . trans($label)
```

Comment:
Chyba jestem przemęczony, albo to "poprawka" powoduje wprowadzenie błędu w sytuacji, gdy $icon zawiera na początku człon `lms-ui-icon-` oraz $icon_class jest niepuste...

Reply:
Zaraz zrobię test

Reply:
Przetestowałem przypadki:
#1: "mail",
#2 "lms-ui-icon-mail", 
#3 "img/mail.gif"
I podesłałem kolejną łatkę.